docs-src/prod-readiness-context/how-to-set-up-codec-server.md

```diff
@@ -203,9 +205,50 @@ However, consider the following before choosing to do so:
 
 After you create your Codec Server and expose the requisite endpoints, set the endpoints in your Web UI and CLI.
 
+#### Web UI
+
+On Temporal Cloud and self-hosted Temporal Clusters, you can set the codec endpoints at a Namespace level in the Web UI.
+
+![Codec Server endpoint setting](/img/docs/data-encoder-button.png)
+
+To set a Codec Server endpoint on a Namespace, do the following.
+
+1. In the Web UI, go to Namespaces, select the Namespace where you want to configure the Codec Server endpoint, and click **Edit**.
+2. In the Codec Server section on the Namespace configuration page, enter your Codec Server endpoint and port number.
+3. [Optional] If your Codec Server is configured to [authenticate requests](#authorization) from Temporal Web UI, enable **Pass access token** to send a JWT access token with the HTTPS requests.
+4. [Optional] If your Codec Server is configured to [verify origins of requests](#cors), enable **Include cross-origin credentials**.
+
+Setting a Codec Server endpoint on a Namespace enables it for all users on the Namespace.
+On Temporal Cloud, you must have [Namespace admin privileges](/cloud/#namespace-level-permissions) to add a Codec Server endpoint on the Namespace.
+
+All users on a Namespace have the option to override the Namespace-level setting at the browser level.
+Overriding the Namespace-level endpoint in your browser affects only your instance of the Web UI (both on Temporal Cloud and self-hosted Cluster).
+This can be useful when developing, testing, or troubleshooting encoding functionality.
+To set a browser override for the Namespace-level endpoint, do the following.
+
+1. Navigate to Workflows in your Namespace.
+2. In the top-right corner, select **Configure Codec Server**.
+3. Select whether you want to use the Namespace-level (or Cluster-level for self-hosted Cluster) or the browser-level Codec Endpoint setting as the default for your instance of the UI.
+   In Temporal Cloud:
+   - Selecting **Use Namespace-level settings, where available. Otherwise, use my browser setting.** uses the Namespace-level Codec Server endpoint by default. If no endpoint is set on the Namespace, your browser setting is applied.
+   - Selecting **Use my browser setting and ignore Namespace-level setting.** applies your browser-level setting by default, overriding the Namespace-level Codec Server endpoint.
```

Comment:
```suggestion
   - **Use Namespace-level settings, where available. Otherwise, use my browser setting.**
      Uses the Namespace-level Codec Server endpoint by default.
      If no endpoint is set on the Namespace, your browser setting is applied.
   - **Use my browser setting and ignore Namespace-level setting.**
      Applies your browser-level setting by default, overriding the Namespace-level Codec Server endpoint.
```